impl/restart_integration_test.go

```diff
@@ -173,15 +176,15 @@ func TestRestartPush(t *testing.T) {
 			}
 
 			// WAIT FOR TRANSFER TO COMPLETE -> THIS SHOULD NOT HAPPEN
-			sentI, receivedI, err := waitF(10*time.Second, 1)
+			sentI, receivedI, err := waitF(2*time.Second, 1)
 			require.EqualError(t, err, "context timed-out without completing data transfer")
 			require.True(t, len(receivedI) < totalIncrements)
 			require.NotEmpty(t, sentI)
+			t.Logf("not request was completed after disconnect")
```

Comment:
Should this be "transfer was not completed after disconnect"?  Didn't quite understand the log